jax/interpreters/pxla.py

```diff
@@ -511,14 +538,26 @@ def __init__(self,
     self.sharding_spec = sharding_spec
     self.indices = indices
     self._npy_value = None
+    self._one_replica_buffer_indices = None
     if not core.skip_checks:
       assert type(aval) is ShapedArray
 
+  @property
+  def one_replica_buffer_indices(self):
```

Comment:
Add a docstring or comment describing this. Also consider making `_one_replica_buffer_indices`, since this probably shouldn't be a public property.